firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -34,17 +34,18 @@ def testCheckAboutPrivateBrowsing(self):
                                                        'p.showNormal')
             self.assertEqual(status_node.text, description)
 
-            access_key = self.browser.get_localized_entity(
-                'privatebrowsingpage.openPrivateWindow.accesskey')
+            # TODO: Augment send_shortcut() to DOMElement and let Marionette return
+            # such an object instead of HTMLElement, so it can be send to the element
 
             # Send keys to the top html element.
-            top_html = self.marionette.find_element(By.TAG_NAME, 'html')
-            top_html.send_keys(self.keys.SHIFT, self.keys.ACCEL, access_key)
+            # top_html = self.marionette.find_element(By.TAG_NAME, 'html')
+            # top_html.send_keys(self.keys.SHIFT, self.keys.ACCEL, access_key)
+            key = 'privatebrowsingpage.openPrivateWindow.accesskey'
+            self.browser.send_shortcut(self.browser.get_localized_entity(key),
+                                       accel=True, shift=True)
 
         self.wait_for_condition(lambda mn: len(self.windows.all) == 2)
-        self.windows.switch_to(lambda win: win.is_private)
-        self.browser_pb = self.windows.current
-
+        self.browser_pb = self.windows.switch_to(lambda win: win.is_private)
         self.assertTrue(self.browser_pb.is_private)
```

Comment:
Why is this assertion redundant? We expect to have a private browsing window, but if something fails in the switch_to() code, we will fail here. So I really do not see why this check is redundant because it tests that switch_to() works as expected.